config/initializers/archive_config/mail.rb

```diff
@@ -21,6 +21,8 @@ class Application < Rails::Application
           address: ArchiveConfig.SMTP_SERVER,
           domain: ArchiveConfig.SMTP_DOMAIN,
           port: ArchiveConfig.SMTP_PORT,
+          enable_starttls_auto: ArchiveConfig.SMTP_ENABLE_STARTTTLS_AUTO,
```

Comment:
There's another `config.action_mailer.smtp_settings` hash in the other if branch:

https://github.com/otwcode/otwarchive/blob/7c86638ec91ca1658f26b77aec55080c3b355d90/config/initializers/archive_config/mail.rb#L10-L19

Do we ever use it? You should either:
- also update that hash with the new options, or
- remove that if branch entirely, along with unused config options `SMTP_USER`, `SMTP_PASSWORD`, `SMTP_AUTHENTICATION` in config.yml.